### ISSUE TITLE
Command-line subcommands don't support --help

### ISSUE THREAD

**Issue by tuukkamustonen** [Miserlou/Zappa#404]
Neither `zappa deploy --help` nor `zappa --help deploy` print out help for that command in specific.

**Comment by skarale25** [comment]
@tuukkamustonen When I run same command, I get following output
$ zappa deploy --help
usage: zappa [-h] [-n NUM_ROLLBACK] [-s SETTINGS_FILE] [-a APP_FUNCTION] [-v]
             [-y] [--remove-logs] [--raw] [--no-cleanup] [--all]
             [U [U ...]]

Zappa - Deploy Python applications to AWS Lambda and API Gateway.

positional arguments:
  U                     Please supply a command to execute. Can be one of:
                        certify, deploy, init, invoke, manage, rollback,
                        schedule, status, tail, undeploy, unschedule, update

optional arguments:
  -h, --help            show this help message and exit
  -n NUM_ROLLBACK, --num-rollback NUM_ROLLBACK
                        The number of versions to rollback.
  -s SETTINGS_FILE, --settings_file SETTINGS_FILE
                        The path to a zappa settings file.
  -a APP_FUNCTION, --app_function APP_FUNCTION
                        The WSGI application function.
  -v, --version         Print the zappa version
  -y, --yes             Auto confirm yes
  --remove-logs         Removes log groups of api gateway and lambda task
                        during the undeployment.
  --raw                 When invoking remotely, invoke this python as a
                        string, not as a modular path.
  --no-cleanup          Don't remove certificate files from /tmp during
                        certify. Dangerous.
  --all                 Execute this command for all of our defined Zappa
                        environments.
Do you get same output? Correct me if I am getting wrong.

**Comment by tuukkamustonen** [comment]
@skarale25 Yeah, that's the output (btw. you should [format your paste](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting)).

**Comment by Miserlou** [comment]
@skarale25 - he wants specific help for each sub command.

**Comment by rozuur** [comment]
Is this open? Can I work on this?
